Commands/Public/e621.js

```diff
@@ -0,0 +1,85 @@
+const { get } = require("snekfetch");
+const S = require("../../Modules/String");
+const PaginatedEmbed = require("../../Modules/MessageUtils/PaginatedEmbed");
+
+module.exports = async ({ Constants: { Colors, NSFWEmbed, Text, APIs, UserAgent } }, { serverDocument }, msg, commandData) => {
+	if (msg.channel.nsfw) {
+		if (msg.suffix) {
+			const m = await msg.channel.send(Text.THIRD_PARTY_FETCH("We're fetching the requested images"));
+			let query = msg.suffix.substring(0, msg.suffix.lastIndexOf(" "));
+			let num = msg.suffix.substring(msg.suffix.lastIndexOf(" ") + 1);
+			if (!query || isNaN(num)) {
+				query = msg.suffix;
+				num = serverDocument.config.command_fetch_properties.default_count;
+			}
+			if (num > serverDocument.config.command_fetch_properties.max_count) {
+				num = serverDocument.config.command_fetch_properties.max_count;
+			}
+			const { body, statusCode } = await get(APIs.E621(query)).set({
+				Accept: "application/json",
+				"User-Agent": UserAgent,
+			});
+			if (body && statusCode === 200 && body.length) {
+				const unparsed = [], descriptions = [], fields = [], images = [];
+				for (let i = 0; i < num; i++) {
+					const random = Math.floor(Math.random() * body.length);
+					if (body[random] && !unparsed.includes(body[random])) unparsed.push(body[random]);
+				}
+				for (let i = 0; i < unparsed.length; i++) {
+					const tempDesc = [
+						`Made by **${unparsed[i].author}**`,
+						`Click [here](https://e621.net/post/show/${unparsed[i].id}) to see the full post.`,
+					];
+					if (unparsed[i].description) {
+						const d = `\`\`\`css\n${unparsed[i].description.split("").slice(0, 1500).join("")
+							.replace(/\[\/?(?:b|u|i)\]/g, "")}${unparsed[i].description.length > 1500 ? `...\`\`\` Read more [here](https://e621.net/post/show/${unparsed[i].id})` : "```"}`;
```

Comment:
How about we remove this `Read more` and add just the click here at the bottom?

Reply:
But... why?

Reply:
Duplicate `Click here to see the full post` tbh..

Reply:
I guess
We would miss some kind of indicator that we truncated though.

Reply:
We still have the `...` at the end of the text